src/MatrixDotNet/Extensions/QudraticForm/Form.cs

```diff
@@ -0,0 +1,12 @@
+namespace MatrixDotNet.Extensions.QudraticForm
+{
+    /// <summary>
+    /// Represents quadratic form.
```

Comment:
But wait, the quadratic form is a polynomial...

![image](https://www.meme-arsenal.com/memes/74c5fcb38966c9184f7ba33644c32609.jpg)


It seems to me that you meant '[type of definite symmetric matrix](https://en.wikipedia.org/wiki/Definite_symmetric_matrix)'